ci/travis/test.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+echo "library path", $LD_LIBRARY_PATH
```

Comment:
Here it is if you like...

https://github.com/opengisch/osgeo4travis/tree/master/docker/qt5

On 09/05/2016 09:27 AM, Hugo Mercier wrote:
> In ci/travis/test.sh
> <https://github.com/qgis/QGIS/pull/3444#discussion_r77479249>:
>
> > @@ -0,0 +1,2 @@
> > +#!/bin/sh
> > +echo "library path", $LD_LIBRARY_PATH
>
> Sure but I don't understand why sqlite does not see it ?
> I should investigate on a docker based on your osgeo4travis
> configuration ...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/qgis/QGIS/pull/3444/files/9f049f5029e10eb0fc54c5ae0e4d60febb4a02fc..20f72320a4c0fb6ebc2dce33ed1ddc9b0322ad41#r77479249>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAj6d8IM7Y7BjRWIPCb3tlyePz1prfSvks5qm8R6gaJpZM4JyiTa>.
>